utilities/db_helper.py

```diff
@@ -29,9 +29,9 @@ def execute_sql_query_with_write(cursor, sql_query, values: tuple):
 
 
 @contextlib.contextmanager
-def open_write_cursor():
-    conn = psycopg2.connect(f"dbname='{Config.DB_NAME}' user='{Config.DB_USERNAME}' host='{Config.DB_HOST_WRITE}' "
-                            f"password='{Config.DB_PASSWORD}' port='{Config.DB_PORT_WRITE}'{Config.DB_USESSL}")
+def open_write_cursor(db_host=Config.DB_HOST, extra_options=""):
```

Comment:
This one needs to default to `Config.DB_HOST_WRITE` instead, it needs to connect to the write instance.

Reply:
`open_write_cursor` is only used by the print batch thingy and it overrides the default to be the action DB